src/main/kotlin/com/zepben/evolve/services/network/translator/NetworkProtoToCim.kt

```diff
@@ -988,6 +1009,12 @@ fun toCim(pb: PBGround, networkService: NetworkService): Ground =
         toCim(pb.ce, this, networkService)
     }
 
+fun toCim(pb: PBGroundingImpedance, networkService: NetworkService): GroundingImpedance =
```

Comment:
todo: Check alphabetical ordering and correct package location for all new calls.

Reply:
moved